### PR TITLE
Hide single-player "fake" teams on Teams index page

### PR DESCRIPTION
**Pull request by bakoe**
See #488

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -78,6 +78,7 @@ def four_different_characters
 
   has_many :team_users
   has_many :teams, through: :team_users, source: :team
+  has_many :real_teams, -> { where single: false }, through: :team_users, source: :team
```
Comment (niconomaa) [inline app/models/user.rb]: maybe naming could be more descriptive than "real" as in "non-single-teams" or such...

**Review by niconomaa — APPROVED** [review]

File: app/models/user.rb (hunk 1 above)
Comment (bakoe) [reply app/models/user.rb]: Good idea—I was not very happy with the name either. I'm gonna change it right now. 🙂

Comment (bakoe) [reply app/models/user.rb]: Resolved in 329929b

File: app/models/user.rb [hunk 2]
```diff
@@ -78,6 +78,7 @@ def four_different_characters
 
   has_many :team_users
   has_many :teams, through: :team_users, source: :team
+  has_many :non_single_teams, -> { where single: false }, through: :team_users, source: :team
```
Comment (jonasnoki) [inline app/models/user.rb:81]: Good idea!!

**Review by jonasnoki — APPROVED** [review]
I like your code 💻

**Pull request merged**